tests/test_TaskMetadata.py

```diff
@@ -383,13 +383,13 @@ def test_filled_metadata_is_filled():
     )
 
 
-def test_all_metadata_is_filled():
+def test_all_metadata_is_filled_and_valid():
     all_tasks = get_tasks()
 
     unfilled_metadata = []
     for task in all_tasks:
         if task.metadata.name not in _HISTORIC_DATASETS:
-            if not task.metadata.is_filled():
+            if not task.metadata.is_filled() and task.metadata.is_validated():
```

Comment:
I wasn't able to find the `is_validated` method. Did you mean `validate_metadata`? Also, might need to wrap the and part in `()` before putting `not` in front 🤔

Reply:
Good catch, apparently there is a pydantic function called the same.